cmake/modules/BuildDPDK.cmake

```diff
@@ -0,0 +1,28 @@
+if (CMAKE_SYSTEM_NAME MATCHES "Linux")
+  if(CMAKE_SYSTEM_PROCESSOR MATCHES "i386")
+    set(dpdk_env  "x86_x32-native-linuxapp-gcc")
+  else()
+    set(dpdk_env  "x86_64-native-linuxapp-gcc")
+  MESSAGE(STATUS "current platform: Linux ")
```

Comment:
again, could you please drop this `MESSAGE()`?

Reply:
ok